src/main/java/fr/techad/edc/popover/utils/OpenUrlAction.java

```diff
@@ -42,19 +94,14 @@ public void openUrl(String url) throws IOException, URISyntaxException {
         LOGGER.debug("Open the url: {}", url);
 
         if(helpConfiguration.getHelpViewer() == HelpViewer.EDC_DESKTOP_VIEWER){
-            Runtime runtime = Runtime.getRuntime();
-            try
-            {
-                if(helpConfiguration.getViewerDesktopPath().isEmpty()){
-                    LOGGER.error("The path of the application must be entered");
-                }else{
-                    runtime.exec(helpConfiguration.getViewerDesktopPath());
+            if(helpConfiguration.getViewerDesktopPath().isEmpty()){
+                LOGGER.error("The path of the application must be entered");
+            }else {
+                setTimeout(() -> System.out.println("test"), 12000, url);
```

Comment:
I don't undestand the goal of this method: wait 12s, display text and fetch an url.